akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -250,6 +253,29 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
       }
     }
 
+    "pass tags from props to MDC" in {
+      val behavior = Behaviors.setup[String] { ctx =>
+        ctx.log.info("Starting up")
+
+        Behaviors.receiveMessage {
+          case msg =>
+            ctx.log.info("Got message {}", msg)
+            Behaviors.same
+        }
+      }
+      val customFilter: LoggingEvent => Boolean = { logEvt =>
```

Comment:
wouldn't it be possible to use `LoggingEventFilter.withMdc`?

Reply:
Ah, good ofc I should use that!